src/widgets/shared-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
+
+export const openWorkspaceTab = (componentToAdd, componentName, params?) => {
```

Comment:
Can you type these params?

Maybe use

```ts
export function openWorkspaceTab(componentToAdd: React.FC, componentName: string, params?: any) {
}
```

- What is `params`
- Is this a type without props? If not, what are these props?
```

Reply:
Sure, params is an Object. It is used for passing around stuff like condition/order uuids, or condition/drug/etc names to the form when editing individual records. I'll close this PR since the changes it contained have already been merged and then open a different one addressing the typing of `params` and `componentToAdd`.

Reply:
> Can you type these params?
> 
> Maybe use
> 
> ```ts
> export function openWorkspaceTab(componentToAdd: React.FC, componentName: string, params?: any) {
> }
> ```
> 
> * What is `params`
> * Is this a type without props? If not, what are these props?

I've addressed these concerns in #58 .